hnn_core/tests/test_lfp.py

```diff
@@ -149,7 +148,8 @@ def test_dipolar_far_field():
         for posy in np.arange(ymin, ymax, step):
             net.add_electrode((posx, posy, posz), sigma=sigma, method=method)
 
-    dpl = simulate_dipole(net, postproc=False)
+    with MPIBackend(n_procs=2):
```

Comment:
Should there be a "requires MPI" decorator here?

Reply:
added